src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -70,6 +70,11 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         bool IsTupleType { get; }
 
         /// <summary>
+        /// True if the type may contain embedded references.
```

Comment:
Can write more here to explain what this actually is to mere mortals?

Reply:
❕ This comment is readily interpreted as a describing non-blittable type, but in reality is something completely unrelated.